src/constant/mysql_types.ts

```diff
@@ -1,60 +1,30 @@
-/** @ignore */
-export const MYSQL_TYPE_DECIMAL = 0x00;
-/** @ignore */
-export const MYSQL_TYPE_TINY = 0x01;
-/** @ignore */
-export const MYSQL_TYPE_SHORT = 0x02;
-/** @ignore */
-export const MYSQL_TYPE_LONG = 0x03;
-/** @ignore */
-export const MYSQL_TYPE_FLOAT = 0x04;
-/** @ignore */
-export const MYSQL_TYPE_DOUBLE = 0x05;
-/** @ignore */
-export const MYSQL_TYPE_NULL = 0x06;
-/** @ignore */
-export const MYSQL_TYPE_TIMESTAMP = 0x07;
-/** @ignore */
-export const MYSQL_TYPE_LONGLONG = 0x08;
-/** @ignore */
-export const MYSQL_TYPE_INT24 = 0x09;
-/** @ignore */
-export const MYSQL_TYPE_DATE = 0x0a;
-/** @ignore */
-export const MYSQL_TYPE_TIME = 0x0b;
-/** @ignore */
-export const MYSQL_TYPE_DATETIME = 0x0c;
-/** @ignore */
-export const MYSQL_TYPE_YEAR = 0x0d;
-/** @ignore */
-export const MYSQL_TYPE_NEWDATE = 0x0e;
-/** @ignore */
-export const MYSQL_TYPE_VARCHAR = 0x0f;
-/** @ignore */
-export const MYSQL_TYPE_BIT = 0x10;
-/** @ignore */
-export const MYSQL_TYPE_TIMESTAMP2 = 0x11;
-/** @ignore */
-export const MYSQL_TYPE_DATETIME2 = 0x12;
-/** @ignore */
-export const MYSQL_TYPE_TIME2 = 0x13;
-/** @ignore */
-export const MYSQL_TYPE_NEWDECIMAL = 0xf6;
-/** @ignore */
-export const MYSQL_TYPE_ENUM = 0xf7;
-/** @ignore */
-export const MYSQL_TYPE_SET = 0xf8;
-/** @ignore */
-export const MYSQL_TYPE_TINY_BLOB = 0xf9;
-/** @ignore */
-export const MYSQL_TYPE_MEDIUM_BLOB = 0xfa;
-/** @ignore */
-export const MYSQL_TYPE_LONG_BLOB = 0xfb;
-/** @ignore */
-export const MYSQL_TYPE_BLOB = 0xfc;
-/** @ignore */
-export const MYSQL_TYPE_VAR_STRING = 0xfd;
-/** @ignore */
-export const MYSQL_TYPE_STRING = 0xfe;
-/** @ignore */
-export const MYSQL_TYPE_GEOMETRY = 0xff;
+export const MYSQL_TYPE_DECIMAL = 0x00
```

Comment:
I don't want the internal constants to be generated in the typedoc documentation. I'm not sure if there is any problem with removing the ignored comments? Or just ignore the entire file?

Reply:
For my taste ignoring every line in a file seems like a hack to me. Since it is only imported in one file, maybe it should be merged with result.ts without exports?